app/views/documents/_attachment.html.erb

```diff
@@ -13,8 +13,18 @@
   # to bust cache and view a specific edition, isn't applicable when publishing
   # to publishing-api.
   preview = controller.nil? ? nil : params[:preview]
+
+  attachment_data = {}
+  if attachment.html?
+    attachment_data[:module] = "ga4-link-tracker"
+    attachment_data[:ga4_track_links_only] = ""
+    attachment_data[:ga4_link] = {
+      "event_name": "navigation",
+      "type": "html attachment"
+    }.to_json
```

Comment:
Readability suggestion:

```Ruby
if attachment.html?
  attachment_data = {
    module: "ga4-link-tracker",
    ga4_track_links_only: "",
    ga4_link: {
      event_name: "navigation",
      type: "html attachment",
    },
  }.to_json
end
```

Reply:
@andysellick Thanks, made the change :+1: